internal/web/endpoint/queue_next.go

```diff
@@ -45,7 +45,12 @@ func QueueNextEndpoint(
 
 			next, err = stateView.ByID(ctx, next.TaskID)
 			if err != nil {
-				return TaskStateResponse{nil, err.Error()}, err
+				switch err.(type) {
+				case taskerrors.NotFoundError:
```

Comment:
Peut-être si c'est Go > 1.13, `if errors.Is(err, taskerrors.NotFoundError) { continue }`, qui supporte les erreurs wrappées dans des messages de contexte (en allant chercher si ça match dans les erreurs sources).